app/controllers/billing_contracts_controller.rb

```diff
@@ -0,0 +1,50 @@
+class BillingContractsController < ApplicationController
+  before_action :set_billing_contract, only: %i[show update edit destroy]
+
+  def index
+    @billing_contracts = BillingContract.all
+  end
+
+  def new
+    @billing_contract = BillingContract.new
+  end
+
+  def show; end
+
+  def create
+    @billing_contract = BillingContract.new(billing_contract_params)
```

Comment:
Enter

Reply:
Where need Enter? If I put write Enter between `def show` and `end`, rubocop will give me message: Put empty method definitions on a single line. (convention:Style/EmptyMethod)